high-score-board/high-score-board.js

```diff
@@ -0,0 +1,81 @@
+/// <reference path="./global.d.ts" />
+
+import { mapPairs } from 'core-js/core/dict';
+
+// @ts-check
+
+/**
+ * Creates a new score board with an initial entry.
+ *
+ * @returns {Record<string, number>} new score board
+ */
+export function createScoreBoard() {
+  const board = {
```

Comment:
You may want to return the object value directly.

```javascript
return {
  'The Best Ever': 1_000_000,
};
```

as opposed to creating the `board` var and returning it down below.